### ISSUE TITLE
Output path settings, per-target

### ISSUE THREAD

**Issue by kylecordes** [BuilderIO/mitosis#566]
## Currently

Mitosis always prepends the target as a directory in the path to the output. This is helpful in some cases where you want to most readily see the output for multiple targets. But it is unhelpful for more sophisticated cases.

## Feature request

**Simplest**: when outputting for only **one target**, don't prepend. Write the output where configured, and no deeper.

**Slightly better**: add an option to control whether the output directory is constructed in this way, in the single-target case.

**Best**: Move/add the target dest configuration to the per-target settings section of the configuration, already used for other per-target settings.

## Workaround

Projects can work around this today by letting the output land where it does, and then copy it to wherever it is needed. This is typically a one-liner with the norm `sync-directory` package.

**Comment by samijaber** [comment]
Yeah, I'm totally open to do this. We could do something where we keep the general `dest` setting, and add per-framework config. And have the `build` CLI go through them by order of priority:

- the framework's specific `dest`
- general `dest`
- default (`output`)
